yaml/pois.yaml

```diff
@@ -26,6 +26,8 @@ global:
     attraction: {col: attraction}
     zoo: {col: zoo}
     exit_to: {col: exit_to}
+  - &health_facility_type_kind_detail
+    kind_detail: {col: tags->'health_facility:type'}
```

Comment:
yamllint.com doesn't like this line at all... or similar tags-> lines with : in the tag key.

Reply:
Yep, CircleCI failed in a similar way on this line.